### ISSUE TITLE
Using `sap.ui.core.Fragment.load` with the `controller`-property

### ISSUE THREAD

**Issue by pubmikeb** [SAP/openui5-docs#97]
While doing a codebase refactoring to get rid of the legacy UI5 APIs, I've faced a need to replace `sap.ui.xmlfragment` with the modern, async API — `sap.ui.core.Fragment.load`.

In the [`sap.ui.core.Fragment`-documentation](https://ui5.sap.com/#/api/sap.ui.core.Fragment%23methods/sap.ui.core.Fragment.load), I've found several snippets how to use this API, e.g.:

```js
sap.ui.require(["sap/ui/core/Fragment"], async function (Fragment) {
	const myFrag = await Fragment.load({
		name: "my.useful.VerySimpleUiPart",
	});
});
```

In my case, this snippet didn't work until I've added `controller: this`, following the @boghyon's [ advice](https://stackoverflow.com/a/61367537) on SO.

In the API documentation, there is a mention of the `controller` property:

> _the Controller or Object which should be used by the controls in the Fragment. Note that some Fragments may not need a Controller while others may need one and certain methods to be implemented by it._

Since in some circumstances the code won't work without the `controller`-property, wouldn't be better to pay more attention to this property in this API's documentation, providing a code snippet and more detailed explanation when and why this property might be necessary, rather than just mentioning _«some Fragments may not need a Controller while others may need one»_?

**Comment by boghyon** [comment]
> didn't work until I've added `controller: this`

Better replace `sap.ui.xmlfragment` with `this.loadFragment`, as shared in ["Methods assigned in XML fragment not triggered" - Stack Overflow](https://stackoverflow.com/a/64541325/5846045), if `this` can be referred to a Controller instance which becomes the event listener of the fragment automatically.

___

API reference: [`sap/ui/core/mvc/Controller#loadFragment`](https://sdk.openui5.org/api/sap.ui.core.mvc.Controller#methods/loadFragment)

**Comment by boghyon** [comment]
> Then, there is no need for an extra call of `sap.ui.require` with loading of `sap/ui/core/Fragment`!

You're right. Internally, [`Controller#loadFragment` requires `sap/ui/core/Fragment`](https://github.com/SAP/openui5/blob/30520023cb15f2ba898732e03e95289908706d01/src/sap.ui.core/src/sap/ui/core/mvc/Controller.js#L863-L866) anyway.

> Then perhaps it's better to mention in the documentation that `this.loadFragment` is the preferable alternative over `sap.ui.core.Fragment.load` to replace `sap.ui.xmlfragment`.

The topic [Instantiation of Fragments](https://ui5.sap.com/#/topic/04129b2798c447368f4c8922c3c33cd7) includes both APIs `loadFragment` & `Fragment.load` with the explanation when to use which one. I'd say it's sufficient for the deprecation notes in `sap.ui.*fragment` APIs to add a link to that topic.

**Comment by KlattG** [comment]
Implemented with UI5 1.132
